app/views/admin/_ab_test_script.html.erb

```diff
@@ -0,0 +1,4 @@
+<script>
+  window.abTestFlag = <%= get_ab_test_flag(alternative_flags: [:onboarding_new, :onboarding_old]).to_json.html_safe %>;
```

Comment:
I'd return all the feature flags that the marketplace has and not only limit it to A/B testing flags. I feel that this can be potentially beneficial information, but more importantly, it would make the code a lot simpler: There wouldn't be need for intersecting and that would make the `get_ab_test_flag` would be trivial \o/